Command.php

```diff
@@ -474,6 +474,18 @@ public function strip(): self
 
         return $this;
     }
+    
+    /**
+     * @see http://imagemagick.org/script/command-line-options.php#trim
+     */
+    public function trim(int $percent_background = 0): self
```

Comment:
As `trim:percent-background` is not mandatory, I suggest either adding two different methods: `trim()` and `trimBackground(int $percentage)` instead, and also validate the `$percentage` value to be between 0 and 100 (to avoid edge cases before running the app, or if the parameter is managed dynamically)

Reply:
IMO It's also worth noting that `-fuzz` is often used conjointedly with `-trim`, so maybe using three different methods would even be better